.github/workflows/wheel_macOS.yml

```diff
@@ -0,0 +1,61 @@
+name: wheel - macOS
+
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  wheel_macos:
+    name: macOS wheel
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest]
```

Comment:
Should we perhaps fix the macos version to mirror the version we are using in `httpstan-wheels`. I think this would be a good idea because changes in macOS versions have been known to break everything (e.g., Catalina).

Reply:
Is there a need to build for different macOS versions? It used to be easy, now macOS binary/compilation stuff is small mystery for me.

Reply:
Can we pick a specific macos version? That's what we do with httpstan-wheels, we use the travis tag "xcode11.3" which uses macOS 10.14.6 (see https://docs.travis-ci.com/user/reference/osx ). Could we perhaps start with the equivalent to that? If you want to change things after we get things working, we could have a new PR.

Reply:
I'm not sure if possible

https://github.blog/changelog/2019-11-06-github-actions-macos-virtual-environment-updated-to-catalina/

Is there possibility for docker?

Reply:
Let's just use `macos-10.15` rather than macos-latest so we know which image we are using and the image used doesn't change unexpectedly when github adds 10.16